public/app/core/utils/datemath.ts

```diff
@@ -80,13 +107,13 @@ export function parseDateMath(mathString, time, roundUp?) {
       return undefined;
     }
 
-    if (isNaN(mathString.charAt(i))) {
+    if (isNaN(parseInt(mathString.charAt(i), 10))) {
```

Comment:
This was not exactly a bug, isNaN actually parses string to number so it works but seems in TS standard library `isNaN` only takes number.